App.tsx

```diff
@@ -53,6 +57,9 @@ export default function App() {
 
         // Add listener for deep links
         await DeepLinkProvider.enableDeepLinking()
```

Comment:
when i created the DeepLinkProvider, the idea was to make it handle all the intents. So i would like to see SharedDataProvider and the DeepLinkProvider to be combined. both use listeners, so that should not be to difficult. And maybe rename it to something like IntentHandler or something similair.

this way we just have 1 place that handles intents

Reply:
and place all the logic fopr intents in this file

Reply:
attempted a fix. I was able to see the WEB share side working. But saw an error in Deep link testing. https://github.com/Sphereon-Opensource/ssi-mobile-wallet/pull/10#discussion_r1042049022

Reply:
Fixed the navigation issue